test error: genComponsTemplate.py

On current git master, I get the following error when trying to run the tests. It looks like ```experiment``` is no longer available in ```psychopy.app```.

```
platform linux2 -- Python 2.7.14, pytest-3.4.0, py-1.5.2, pluggy-0.6.0
rootdir: /home/damien/venv_study/psychopy/psychopy, inifile: pytest.ini
collected 680 items / 1 errors                                                                                                                               

=========================================================================== ERRORS ===========================================================================
________________________________________ ERROR collecting psychopy/tests/test_app/test_builder/genComponsTemplate.py _________________________________________
test_app/test_builder/genComponsTemplate.py:13: in <module>
    from psychopy.app import builder, projects, experiment
E   ImportError: cannot import name experiment
```

--- a/psychopy/tests/test_app/test_builder/genComponsTemplate.py
+++ b/psychopy/tests/test_app/test_builder/genComponsTemplate.py
@@ -10,7 +10,8 @@ if parse_version(wx.__version__) < parse_version('2.9'):
     tmpApp = wx.PySimpleApp()
 else:
     tmpApp = wx.App(False)
-from psychopy.app import builder, projects, experiment
+from psychopy import experiment
+from psychopy.app import builder, projects
 from psychopy.experiment.components import getAllComponents
 
 # usage: generate or compare all Component.param settings & options

